src/index.html

```diff
@@ -4,9 +4,405 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/general.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#HOME">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a class="header__nav-link is-active" href="#APPLE">apple</a>
+          </li>
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#SAMSUNG">samsung</a>
+          </li>
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#СМАРТФОНЫ И ТЕЛЕФОНЫ">смартфоны и телефоны</a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#НОУТБУКИ И КОМПЬЮТЕРЫ"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#ГАДЖЕТЫ">гаджеты</a>
+          </li>
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#ПЛАНШЕТЫ">планшеты</a>
+          </li>
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#ФОТО">фото</a>
+          </li>
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#ВИДЕО">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="card-conteiner">
```

Comment:
```suggestion
      <div class="cards-container">
```
Опечаточка :)
Ну, а ещё - что бы не путаться, лучше называть контейнеры, с большим количеством однотипных блоков в множественном числе. Так сразу понятно, что это контейнер ВСЕХ карточек, а не только одной

Reply:
Как вариант, спасибо)

Reply:
+